crates/polars-core/src/datatypes/any_value.rs

```diff
@@ -1708,6 +1710,12 @@ impl From<bool> for AnyValue<'static> {
     }
 }
 
+impl From<NaiveDate> for AnyValue<'static> {
```

Comment:
Can we actually use this code somewhere in Polars. 

If we add it, we should also replace the code where we did this manually.